index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <a href="#">My Logo</a>
+        
```

Comment:
- [ ] You have some unnecessary blank lines in your code. Kindly remove all the unnecessary blank lines.

Reply:
> * [ ]  You have some unnecessary blank lines in your code. Kindly remove all the unnecessary blank lines.

- [ ] Kindly work on it. :+1: